docs/installation/docker/converting.md

```diff
@@ -0,0 +1,87 @@
+# Converting ISLE-DC to Site Template
+
+ISLE-DC is being deprecated in favour of using the Isle Site Template for Docker installs of Islandora. The following instructions explain how to convert your ISLE-DC site into the Site Template style of docker-compose.yml.
```

Comment:
I just want to double check in the tech call the wording of this first sentence. Otherwise, I think this is good to merge. Thanks Josh!